src/app/components/client/ComboBox.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../tokens";
+
+.comboBox {
+  background: $color-white;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  .inputField {
+    border: 1px solid $color-grey-30;
+    border-radius: $border-radius-sm;
+    color: $color-black;
+    padding: calc($spacing-sm * 1.5) $spacing-md;
```

Comment:
suggestion (nit): Add note about calculated number

Reply:
Added a note in https://github.com/mozilla/blurts-server/pull/3217/commits/eb18b2c13c5c0ff4120313ff32740e32c1552da9.